birdhouse/optional-components/README.md

```diff
@@ -112,3 +112,26 @@ Magpie will be automatically configured to give complete public anonymous
 access for this WPS service.
 
 Canarie monitoring will also be automatically configured for this WPS service.
+
+
+## All access given to test user
+
+Can be used for testing purposes.
+Allows the tests in https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests to pass with the `authtest/authtest` user.
```

Comment:
Remove all mentions of the `authtest` user and instructions to create it, I've create a script to do it in my other PR.  For security reasons, let's not broadcast too loudly this hardcoded user with known hardcoded password.